Automattic/wp-calypso
# Jetpack Partner Portal: implement URL support to show more info modal about the license in the issue license page

#78955 had some failing pre-release e2e tests we didn't catch earlier. A URL parsing error caused it. The full discussion can be found here: p1688975846269469-slack-C02DQP0FP

Related to 1202619025189113-as-1204953498074085

![mobile (23)](https://github.com/Automattic/wp-calypso/assets/10586875/a79f2eb2-036b-4aa9-898b-00ea0ddacfca)

## Proposed Change
This PR modifies the logic to use the existing `getQueryArg` helper function to extract the **page** query param. Please see the exact changes on this [commit](https://github.com/Automattic/wp-calypso/pull/79173/commits/bdbd97d4998693570a289b1f3b09e022065ce1c6).

## Testing Instructions
To test the original PR. please refer to #78955 test instructions.

Additionally please test the Videopress setup flow where the issue was caught.
* Use the Calypso live link below and append `/setup/videopress` to start the setup
* Fill in some site information to complete the setup.
* the page should redirect to the checkout page at the end of the setup.
* Confirm that the checkout page is rendered properly without any issues.

## Pre-merge Checklist
